File: test/ocamlTests/BoolTest/equalTest.ml
test "equal(true,true)" (fun () -> expect (Bool.equal true true) |> toEqual Eq.bool true) ; 
test "equal(false,false)" (fun () -> expect (Bool.equal false false) |> toEqual Eq.bool true) ; 
test "equal(true,false)" (fun () -> expect (Bool.equal true false) |> toEqual Eq.bool false) ; 

File: test/fsharpTests/BoolTest/equalTest.fs
testCase "equal(true,true)" 
<| fun _ -> 
    let expected = true
    Expect.equal expected (Bool.equal true true) "error"
testCase "equal(false,false)" 
<| fun _ -> 
    let expected = true
    Expect.equal expected (Bool.equal false false) "error"
testCase "equal(true,false)" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Bool.equal true false) "error"
